### DISCUSSION TITLE
disk partitioning

### DISCUSSION THREAD

**Discussion by kontox** [siderolabs/talos#4322] [Q&A] [answered]
Hi All,

Is it possible to say in the config to leave a part of the disk unformated or just create a partition but not format or mount it? I have a small PC with a 120GB disk and want to use it as a test machine. The idea is to have 20GB for Talos and 100GB for Ceph. I know it's not best to have Ceph OSD on the same disk as the OS but this is just for a test machine. 

There is a disk section in the config but it seems that you have to specify a mountpoint which then wouldn't work with Ceph as far as I know. Any ideas if this can be done? 

Thamls

**Comment by smira** [comment, marked as answer]
Hi @kontox.

At the moment the way Talos operates it assumes ownership of the system disk, so it's not possible to do that.

Even though it's not supported, but you could modify partition scheme once Talos is installed (with external system/live CD of some OS). Talos might blow it up on upgrade though.

**Reply by kontox** [reply]
thanks a lot for the answer, I'll connect an external disk and use that for ceph.

**Comment by Dzeri96** [comment]
I agree that this would be a very useful feature.

**Comment by ffetech** [comment]
Partitioning would be a great feature.

My hosting provider (Netcup) does not support attaching a second fast ssd. Only slowly cloud storage is possible. I guess it‘s the same by hetzner for instance.

**Comment by wokalski** [comment]
In our case, Talos is no-go only because of this. Our machines have 4x1.9TB disks...

**Comment by steverfrancis** [comment]
This feature request is discussed in https://github.com/siderolabs/talos/issues/4041

**Reply by idc77** [reply]
Yeah it's closed as not planned

**Reply by onedr0p** [reply]
If you would have read thru the issue you would have seen it's here now https://github.com/siderolabs/talos/issues/8367
